packages/headless-react/jest.config.js

```diff
@@ -1,8 +1,14 @@
 /** @type {import('ts-jest').JestConfigWithTsJest} */
 module.exports = {
-  preset: 'ts-jest',
   testEnvironment: 'jsdom',
-  collectCoverage: true,
+  transform: {
+    '^.+\\.tsx?$': [
```

Comment:
I think there may be 1-too-many `\` characters in this regex... That is unless you're _actually_ expecting files to be of the format `my-filename\.tsx` rather than `my-filename.tsx`, in which case you can ignore me hehe

Reply:
The double \ is because the regex is in a string: you need to escape `\` by having `\\`